JS/company.js

```diff
@@ -0,0 +1,75 @@
+const urlParams = new URLSearchParams(window.location.search);
+const companySymbol = urlParams.get("symbol");
+const baseURL = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/`;
```

Comment:
baseURL is alredy decleaerd in main.js use another file for shared logic (variables and functions)